### PR TITLE
UpdateRepos

### PR DESCRIPTION
**Pull request by suhasj**
@pranavkm 

I have updated the list of repos. Let me know if I missed any

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -421,9 +426,10 @@ functions
             var nonDefaultRepos = new[]
             {
               "KRuntime",
-              "MusicStore"
+              "MusicStore",
+              "Coherence"
```
Comment (pranavkm) [inline makefile.shade]: Also needs to include `Coherence-Signed` and `dnvm`

**Comment by pranavkm** [comment]
:shipit:

File: makefile.shade (hunk 1 above)
Comment (suhasj) [inline makefile.shade]: Coherence-signed doesn't have Nuget.config so I might need to special case it. I'll add dnvm

File: makefile.shade [hunk 2]
```diff
@@ -421,9 +426,10 @@ functions
             var nonDefaultRepos = new[]
             {
               "KRuntime",
```
Comment (Praburaj) [inline makefile.shade]: Should we call this XRE?

**Comment by Praburaj** [comment]
why not for beta4 (to release branch)?

File: makefile.shade (hunk 2 above)
Comment (suhasj) [inline makefile.shade]: Github routes it XRE anyway but sure I'll change it

**Comment by suhasj** [comment]
we have already branched to beta4 so this is for the next iteration

**Comment by Praburaj** [comment]
Thanks :shipit:

**Comment by victorhurdugaci** [comment]
I'm going to close this since it is so old. Please reopen if you still think it is necessary

**Pull request closed without merging**